src/xunit.performance.api/XunitPerformanceHarnessOptions.cs

```diff
@@ -15,6 +15,7 @@ internal sealed class XunitPerformanceHarnessOptions
         public XunitPerformanceHarnessOptions()
         {
             _outputDirectory = Directory.GetCurrentDirectory();
+            _temporaryDirectory = Path.GetTempPath();
```

Comment:
Just want to make sure that we clean this up when the run completes, right?  It's quite possible for these scenarios to pull down lots of stuff and we could fill up hard drives quickly.